### PR TITLE
Add Rancher Desktop info

### PR DESCRIPTION
**Pull request by bamurtaugh**
Add info about Rancher Desktop to the Docker options doc.

### CODE REVIEW

**Comment by Chuxel** [comment]
I think we'd probably say Rancher Desktop is another way to install docker, so it can likely move up into the sections on that.  While it does support nerdctl as well, their guide routes people to the docker CLI.  Otherwise looks good!

**Review by Chuxel — APPROVED** [review]

**Comment by bamurtaugh** [comment]
Thanks @Chuxel, makes sense! 

I moved it to the intro section overviewing alternate ways to install Docker.

@gregvanl This is ready for your review and merge.

File: remote/advancedcontainers/docker-options.md [hunk 1]
```diff
@@ -16,6 +16,8 @@ You can use Docker with the [Remote - Containers](https://marketplace.visualstud
 * Docker installed on another machine or remote environment.
   * You only need Docker installed on the remote host, rather than Docker installed locally.
 * Other Docker compliant CLIs, installed locally or in a remote environment.
+  * For instance, [Rancher Desktop](https://docs.rancherdesktop.io/) is another way to install Docker, providing container management and Kubernetes on Windows, macOS, and Linux.
```
Comment (gregvanl) [inline remote/advancedcontainers/docker-options.md:19]: Should we add a link to https://mobyproject.org/ for the Moby reference?

Comment (bamurtaugh) [reply remote/advancedcontainers/docker-options.md:19]: Sounds good to me! I added a link.

**Pull request merged**